annotator-core/src/test/java/edu/ucr/cs/riple/core/DeepTest.java

```diff
@@ -87,6 +87,39 @@ public void field_assign_nullable_bailout_disabled() {
         .start();
   }
 
+  @Test
+  public void testTwoReports() {
```

Comment:
What are we testing here that wasn't the case before this PR? I am not sure I follow.

Reply:
This is specifically designed to make sure we do not count an error for a constructor failing to initialize `@Nonnull` fields multiple times.

In the example below and in the unit test:
```java
class Main{
      Object field1;
      Object field2;
      Main() {
          field1 = null;
          field2 = null;
      }
}
```
Here we have an error from the constructor that is failing to initialize `field1` and `field2`. If we annotate `field1` as `@Nullable` we will still get the error that constructor is failing to initialize `field2`. We shouldnt double count this as a new error and include its resolving fixes in the tree.


Renamed unit test f8946f4abb76f65e087198648fd11e8d16dcbb84